WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -237,6 +254,23 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
                         }
                     }
             );
+
+            mTags = (SuggestionAutoCompleteText) mRootView.findViewById(R.id.tags);
+            if (mTags != null) {
+                mTags.setTokenizer(new SuggestionAutoCompleteText.CommaTokenizer());
+
+                remoteBlogId = -1;
```

Comment:
Rather than have `remoteBlogId` as a class variable, would it make more sense to have it as a method variable that's passed to `setupSuggestionServiceAndAdapter()`?